agile_problem2.exs

```diff
@@ -0,0 +1,32 @@
+defmodule Problem2 do
+	def largest_2_numbers(), do: -2
+	
+	def largest_2_numbers(list) do
+		#First check that we have at least 2 elements. Otherwise it's useless
+		if Enum.count(list)<2 do
+			-1
+		else
+			#make the list have unique values.
+			ulist = Enum.uniq(list)

```

Comment:
This is a inefficient way to solve this problem, `uniq` is probably O(log n) and `max` is O(n). You can solve this with O(n).

Not sure why you used unique, the 2 largest numbers in [1, 2, 2] are 2 and 2, it happens that they are the same but that is because of the input, the problem did not stated to find the unique max.